mlir/include/mlir/Dialect/Affine/Passes.td

```diff
@@ -48,6 +48,12 @@ def AffineLoopInvariantCodeMotion
   let constructor = "mlir::createAffineLoopInvariantCodeMotionPass()";
 }
 
+def AffineLoopInterchange
+    : FunctionPass<"affine-loop-interchange">{
+  let summary = "Performs loop interchange of all affine loop nests";
```

Comment:
Mention loop interchange for what? Locality, parallelism?

Reply:
Done. Thanks.